website/integrations/services/oracle-cloud/index.md

```diff
@@ -46,3 +46,76 @@ Create a new _Social IdP_ via the _Add IdP_ button. Set the name to authentik an
+
+Download the XML Document behind the link behind __Download this document__.
+
+### Federation Method, Step 2 - authentik
+
+In authentik, under _Providers_, create an _SAML Provider from Metadata_ with these settings:
+
+-   Name: Oracle Cloud SAML <your tenancy> (or whatever you like as name for this provider)
+-   Authorization flow: default-provider-authorization-explicit-consent (Authorize Application), implicit-consent may work too.
+-   Metadata: Browse to the downloaded XML Document and 
+
+Click Finish. Edit the Provider, below Protocol settings:
+
+-   ACS URL: Copy this URL
+-   Audicence: If empty, paste the ACS URL
+
+Click Update. Click on the name of the new provider in the list to see the overview. Use the link _Create Application_ on this page to create the application for the provider.
+
+Create an application which uses this provider. Optionally apply access restrictions to the application using policy bindings.
+
+-   Name: Oracle Cloud SAML <your tenancy> (or whatever you like as name for this application)
+-   Slug: oracle-cloud-saml-<your tenancy> (or whatever yoou like)
+-   Group: You may want to enter a group name like Cloud-Admins (and create it) or use the Administriators group, note it for later.
+-   Provider: Select the provider you created from the list.
+
+Click Create. Back on the overview page page of the SAML provider, you show now see new information blocks.
+
+The 2nd is titled __Related objects__ below the title, the label __Metadata__, and below it a __Download__ button. Download the Metadata using this button.
+
+### Federation Method, Step 3 - Oracle Cloud
+
+In Oracle Cloud, open the top-left navigation and go to _Identity & Security_ and then _Federation_.
+
+In the title bar of the list of Identity Providers, you should see the button "Add Identity Provider"
+
+Name: The name you enter here will be shown to the users when they select an Identity Provider from the list at long
+Description: Whatever you like.
+Type: SAML 2.0 Compliant Identity Provider
+Upload the FederationMetadata.xml document from your Active Directory Federation Services server: Upload or drop the Metadata file downloaded from the authentik provider's Overview page.
+Authentication Context Class References: Don't select any, as Authentik provides different references that are not listed here. If they do not match, the user gets an XML error showing both.
+
+Click Continue
+
+Here you’ll map groups defined in your Identity Provider to groups defined in Oracle Cloud Infrastructure (OCI). Each group can be mapped to one or more groups of the other kind.
+
+Identity Provider Group: If you entered a Group for this provider in Authentik, you may have to use the same same Group here.
```

Comment:
This is weird. The group set in the application is only used for display purposes (grouping applications together in the user interface).

Am I missing something?